tests/sqlparser_postgres.rs

```diff
@@ -988,6 +988,7 @@ fn parse_create_schema_if_not_exists() {
         Statement::CreateSchema {
             if_not_exists: true,
             schema_name,
+            ..
```

Comment:
I personally think adding explicit ignores here

```suggestion
            options: _, 
            default_collate_spec: _,
```

Is better as then if a new field is added then the compiler will point out this place as needing to be checked

That being said, I don't think it is critical in this case

Reply:
Done!